terraform-docker/apply/main.yaml

```diff
@@ -52,6 +61,55 @@ stages:
                         artifact: "${{ env.name }}.enc"
                         path: $(Build.Repository.LocalPath)/${{ parameters.terraformFolder }}/.terraform/plans
 
+                    - task: AmazonWebServices.aws-vsts-tools.AWSShellScript.AWSShellScript@1
+                      displayName: "Get AWS credentials"
+                      inputs:
+                        workingDirectory: "."
+                        awsCredentials: ${{ format(parameters.awsServiceConnectionTemplate, env.name) }}
+                        regionName: ${{ parameters.awsRegion }}
+                        scriptType: inline
+                        inlineScript: |
+                          set -e
+                          AWS_CONFIG_FILE=${HOME}/.aws/config
```

Comment:
There's a temp variable for Azure DrvOps that maybe can be used instead of HOME?

Reply:
I follow the same way that have been done in azure.
```
                    AZURE_CONFIG_DIR="${AZURE_CONFIG_DIR:-${HOME}/.azure}"
                    mkdir -p ${PWD}/${DIR}/.terraform/
 ```

We can change but then I would like to change in both the places.
Personally I think it's okay like this and if we want to change it we can do it in a separate PR.
Sound okay?